deploifai/project/create.py

```diff
@@ -212,23 +212,14 @@ def prompt_gcp_key():
 
             gcp_service_account_key_file = prompt_gcp_key()
 
-            if gcp_service_account_key_file[0] == '/':
-                while True:
-                    try:
-                        cloud_credentials["gcpServiceAccountKey"] = open(gcp_service_account_key_file, 'r').read()
-                        break
-                    except FileNotFoundError as err:
-                        click.secho("File not found. Please input the correct file path.",fg="red")
-                        gcp_service_account_key_file = prompt_gcp_key()
-            else:
-                while True:
-                    try:
-                        cloud_credentials["gcpServiceAccountKey"] = open(
-                            os.path.join(os.getcwd(), gcp_service_account_key_file), 'r').read()
-                        break
-                    except FileNotFoundError as err:
-                        click.secho("File not found. Please input the correct file path.", fg="red")
-                        gcp_service_account_key_file = prompt_gcp_key()
+            while True:
+                try:
+                    with open(gcp_service_account_key_file) as gcp_service_account_key_json:
+                        cloud_credentials["gcpServiceAccountKey"] = json.load(gcp_service_account_key_json)["private_key"]
```

Comment:
not sure about this one, but I saw backend and this `gcpServiceAccountKey` is asking for string, not sure if I should parse in the whole json or the `private_key` field only from the `service-account-key.json`

Reply:
read the file content as a string and pass it to `gcpServiceAccountKey` in the argument